setsubun.p8

```diff
@@ -79,21 +80,59 @@ function credits_scn(nxt)
 	return scn
 end
 
+function spawn_beans(beans)
+	while true do
+		-- wait 5 frames
+		for i=1,5 do
+			yield()
+		end
+		-- then spawn a new bean
+		local x=rnd(128)
+		local y=128+28+rnd(64)
+		local z=0
+		local b=bean_new(
+			v3(x,y,z),
+			v3(0,0,0)
+		)
+		add(beans, b)
+	end
+end
+
 function title_scn(nxt)
-	local scn = {}
+	local scn={}
+	local beans={}
+	local make_beans=cocreate(spawn_beans)
 
 	function scn.init(s)
 		music(0)
 	end
 
 	function scn.update(s,dt)
+		if costatus(make_beans) then
+			coresume(make_beans, beans)
```

Comment:
I feel like you should allow the coroutine to end once we are past the title screen, otherwise it just sits in memory. Could do one last coresume(nil) before transitioning to the next scene, and have the coroutine return when beans is nil?